tests/test_trigger.py

```diff
@@ -1,80 +1,71 @@
-import unittest
-
+import pytest
 from aiounittest import async_test
 
 from Modules.trigger import Trigger
 from tests.mock_bot import MockBot
 
+@pytest.fixture
+def bot():
+    return MockBot()
+
+def test_create_manual_channel(bot):
+    trigger = Trigger(bot, ["don't", "care"], ["don't care", "care"], "test_nick", "#somechannel",
+                      "test_ident", "test.vhost")
+    assert trigger.channel == "#somechannel"
 
-class TriggerTests(unittest.TestCase):
```

Comment:
I  would appreciate this still being in a class, for the sake of test organization, but change it to inherit from `Object` rather than `TestCase`

(consistency / nitpick)

Reply:
I don't know... This doesn't need to be object-oriented so I made it not be...

As for consistency, we are already using two separate test frameworks ;)

Reply:
We should probably convert all the old to the new at some point 😉 

I was using classes more for Struture than anything else.. its a minor nit pick

Reply:
Yeah, a lot of people have a lot of opinions on where and to what degree code should be object-oriented.

I personally believe it's great and powerful, but is often overused.

Reply:
Well, i guess its vestigial, a left over from using `unittests` for testing as it required tests be instance methods of a class based on `unittest.TestCase`.